Automattic/wp-calypso
# Stats: Trying to add The Last updated time to the StatsModule component

closes #10911 

Since the stats data are now persisted in the local cache, when first hitting the stats page, we show some outdated stats while the query is fetching. So in this PR, I'm adding a blinking effect to the header of the stats module component while the query is fetching.

Also, I added a "time" icon with a title showing the elapsed time since the last stats request. 
All this is experimental and needs a design review.

<img width="522" alt="screen shot 2017-01-30 at 14 11 07" src="https://cloud.githubusercontent.com/assets/272444/22424238/fa3bf14a-e6f5-11e6-95a2-96da3da40f92.png">

**Testing instructions**

 - Open a stats page: `stats/day/$site`
 - Notice the "time" icon on the header of the stats panels
 - Also, notice the blinking effect on the header while the query is refreshing
